src/DAS-Capture-The-Flag.Application/Models/GameModels/Coordinates.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace DAS_Capture_The_Flag.Hubs
+{
+    public class Coordinates
+    { 
+        public int X { get; set; }
+        public int Y { get; set; }
+    }
+}
```

Comment:
Take a look at the [Point struct](https://docs.microsoft.com/en-us/dotnet/api/system.drawing.point)  - your re-inventing the wheel :)